runtime/v2/vm/src/main/java/com/walmartlabs/concord/svm/MultiException.java

```diff
@@ -28,9 +29,10 @@
 public class MultiException extends RuntimeException {
 
     private static final long serialVersionUID = 1L;
+    private static final int MAX_STACK_TRACE_ELEMENTS = 3;
 
     public MultiException(Collection<Exception> causes) {
```

Comment:
maybe rename `MultiException` to `ParallelExecutionException` ?
so that it fits with `"Parallel execution errors: ...` message

Reply:
Good idea!